scipy/special/_generate_pyx.py

```diff
@@ -1295,10 +1296,8 @@ def generate_ufuncs(fn_prefix, cxx_fn_prefix, ufuncs):
                 cxx_defs.extend(item_defs)
                 cxx_defs_h.extend(item_defs_h)
 
-                cxx_defs.append("cdef void *_export_{} = <void*>{}".format(
-                    var_name,
-                    ufunc.cython_func_name(c_name, specialized=True, override=False)
-                ))
+                cxx_defs.append(f"cdef void *_export_{var_name} = "
+                                f"<void*>{ufunc.cython_func_name(c_name, specialized=True, override=False)}")
```

Comment:
Is this correct? If so, hopefully the linter will be happy!
```suggestion
                func_name = ufunc.cython_func_name(c_name, specialized=True, override=False)
                cxx_defs.append(f"cdef void *_export_{var_name} = <void*>{func_name}")
```

Reply:
Linter finally passes! Hooray! Thanks for the suggestion, @lucascolley.